integreat_cms/cms/views/pages/page_actions.py

```diff
@@ -771,20 +771,17 @@ def get_page_order_table_ajax(request, region_slug, parent_id=None, page_id=None
 
     region = request.region
 
-    if page_id:
-        page = get_object_or_404(region.pages, id=page_id)
+    page = get_object_or_404(region.pages, id=page_id) if page_id else None
+
+    if parent_id:
+        parent = get_object_or_404(region.pages, id=parent_id)
+        siblings = [
+            sibling
+            for sibling in parent.cached_children
+            if not sibling.explicitly_archived
+        ]
     else:
-        page = None
-
-    if parent_id or page:
-        parent = (
-            get_object_or_404(region.pages, id=parent_id) if parent_id else page.parent
-        )
-        siblings = parent.cached_children
```

Comment:
@david-venhoff do you remember why you added the [`or page` case](https://github.com/digitalfabrik/integreat-cms/commit/7c505dab0e123302eab5fee49c0ad6a6a6478e94#diff-4956f040b51352b8fde20d1ff3cd57f3879253563dd16d10a75abb5550915cd4) in  in PR #1463?
I believe this is the source of the bug - since it always loads the "old" siblings of the page when no `parent_id` is given (but in this case we want the root pages of the region since we want to move the page to a new location)...
Or am I missing something?

Reply:
I think at some point there was a bug that the parent page input would default to "-----", while the page order table showed all the pages siblings by default (as it still does). Then, when changing the parent page input to something and then going back to "-----", the page order table showed something different. I fixed that inconsistency in the pr not realizing that "-----" was supposed to mean "no parent page" instead of "default". 
So from my point of view there are no objections to this change, this seems much better :+1:

Reply:
Alright, thanks for the feedback!